src/test/java/org/candlepin/dto/api/v1/EnvironmentTranslatorTest.java

```diff
@@ -66,6 +66,7 @@ protected Environment initSourceObject() {
 
         source.setId("test_id");
         source.setName("test_name");
+        source.setName("custom");
```

Comment:
I'm guessing this is intended to be `setType`?

Reply:
Yes. Done.